server/startup/es6.js

```diff
@@ -58,8 +58,8 @@ var WeakMap = new 'WeakMap';
   var struct = [
     [Object, 'Object', ['is', 'setPrototypeOf'], []],
     [String, 'String', [], ['endsWith', 'includes', 'repeat', 'startsWith']],
-    [Number, 'Number', ['isFinite', 'isNaN', 'isSafeInteger'], []],
-    [Math, 'Math', ['sign', 'trunc'], []],
+    [Number, 'Number', ['isFinite', 'isInteger', 'isNaN', 'isSafeInteger'], []],
+    [Math, 'Math', ['acosh', 'asinh', 'atanh', 'cbrt', 'clz32', 'cosh', 'expm1', 'fround', 'hypot', 'imul', 'log10', 'log1p', 'log2', 'sign', 'sinh', 'tanh', 'trunc'], []],
```

Comment:
We ought to fix the line length issues in the startup files, but perhaps in a separate PR.

Reply:
Fixed.